lib/_http_client.js

```diff
@@ -297,6 +310,24 @@ function ClientRequest(input, options, cb) {
 Object.setPrototypeOf(ClientRequest.prototype, OutgoingMessage.prototype);
 Object.setPrototypeOf(ClientRequest, OutgoingMessage);
 
+Object.defineProperty(ClientRequest.prototype, 'aborted', {
+  get() {
+    return this[kWritableState].destroyed;
```

Comment:
The aborted property should likely be marked deprecated also, but it's not critical

Reply:
documentation deprecation?

Reply:
added deprecation